### PR TITLE
[ID] How You Can Help!

### PR DESCRIPTION
**Pull request by fajar13k**
### Description

- Add Bahasa Indonesia localisation on `How You Can Help` article

### Status

- [x] Pending Review

### CODE REVIEW

File: wiki/How_You_Can_Help!/id.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Berikut adalah daftar terkini hal-hal yang saya pikir semua orang dapat membantu, disortir berdasarkan dari yang paling penting di mata saya. Saya akan mencoba terus memperbarui kiriman ini ketika saya melihat hal-hal lain terpikirkan, tetapi jangan ragu untuk memberikan saya (sebaiknya di kiriman ini) jika Anda memilikinya).
+
+## Bantu pengembangan [osu!wiki](https://github.com/ppy/osu-wiki "osu-wiki di GitHub")!
+
+Untuk menjaga konten di situs web tetap segar dan terkini, saya ingin memindahkan semua bagian yang berisi teks ke wiki. Hal ini termasuk FAQ, panduan ranking (ranking guidelines), semua panduan berbasis forum kami, bagian sejarah/informasi. Meskipun kami telah membuat kemajuan besar (berkat beberapa pengguna yang sangat berdedikasi), memperbarui wiki adalah proses berkelanjutan yang pada dasarnya dapat dilakukan oleh siapa pun. Ada juga banyak kesempatan untuk melakukan terjemahan yang tersisa untuk membuat konten dapat diakses dalam bahasa lain yang dikenal umat manusia!
```
Comment (Rendyindo) [inline wiki/How_You_Can_Help!/id.md]: "tetap segar dan terkini" -> "sesegar mungkin"? kayaknya dua kata "as possible" ganjal dengan "dan terkini"

File: wiki/How_You_Can_Help!/id.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+dan yang terakhir, tetapi tidak terlewatkan:
+
+## Hindari berpikiran negatif! Buat orang bahagia!
+
+Setiap hari saya melihat hal negatif di osu!. Orang-orang mengeluh antara satu sama lain, orang-orang mengeluhkan pemain yang curang atau "rank farmers". Orang-orang melihat perubahan dan merengek tentang hal itu. Sementara saya pikir kita merupakan komunitas yang lebih baik dibanding komunitas online lainnya di luar sana - yang turun ke tingkat mulai menghina satu sama lain sepanjang hari (hi youtube!) - masih ada celah yang dapat diperbaiki.
+
+### Dimulai dari mana:
+
+-   Bersikap ramah satu sama lain. Hidup terlalu pendek untuk diperdebatkan.
+-   Don't complain about cheaters. Just [report them and get on with your day](/wiki/Reporting_Bad_Behaviour/Handling_Cheating "Handling Cheating/Foul Behaviour"). They may have taken your top rank, but a simple "!report <username> seems to be cheating" can fix this in minutes to hours.
```
Comment (Rendyindo) [inline wiki/How_You_Can_Help!/id.md]: wait a min why is this untranslated

File: wiki/How_You_Can_Help!/id.md (hunk 1 above)
Comment (fajar13k) [reply wiki/How_You_Can_Help!/id.md]: sesegar mungkin seems like a drink not an article.
let's wait another reviewer

File: wiki/How_You_Can_Help!/id.md (hunk 2 above)
Comment (fajar13k) [reply wiki/How_You_Can_Help!/id.md]: ffs i forgot to paste that section ;_;
lemme fix

**Pull request merged**